### PR TITLE
Name some misc. symbols.

### PR DESCRIPTION
**Pull request by Meowsy**
This is mostly just renaming empty functions to be `[Class]_DoNothing`. The rest of the changes are just some low-hanging fruit I felt relatively confident in.

### CODE REVIEW

File: src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c [hunk 1]
```diff
@@ -80,7 +80,7 @@ static InitChainEntry sInitChain2[] = {
     ICHAIN_F32(uncullZoneDownward, 700, ICHAIN_STOP),
 };
 
-static BgSpot18ObjInitFunc D_808B913C[] = {
+static BgSpot18ObjInitFunc sInitFuncs2[] = {
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c:83]: then this can drop the 2
```suggestion
static BgSpot18ObjInitFunc sInitFuncs[] = {
```

**Review by Dragorn421 — CHANGES_REQUESTED** [review]
Thanks for your contribution! some comments
(not giving you the cold shoulder in comments, we just usually get to the point on reviews without pleasantries or it would take ages)

File: src/overlays/actors/ovl_En_Ik/z_en_ik.c [hunk 2]
```diff
@@ -1383,7 +1383,7 @@ void func_80A77EDC(EnIk* this, GlobalContext* globalCtx) {
     CLOSE_DISPS(gfxCtx, "../z_en_ik_inConfrontion.c", 653);
 }
 
-static EnIkDrawFunc sDrawFuncs[] = { func_80A77ED0, func_80A77EDC, func_80A77844 };
+static EnIkDrawFunc sDrawFuncs[] = { EnIk_DoNothing, func_80A77EDC, func_80A77844 };
```
[the remark this answers is not in the document]
Comment (Meowsy) [reply src/overlays/actors/ovl_En_Ik/z_en_ik.c:1386]: Whoops, missed that one.

**Comment by Meowsy** [comment]
> Thanks for your contribution! some comments (not giving you the cold shoulder in comments, we just usually get to the point on reviews without pleasantries or it would take ages)
No worries; I'm bad at pleasantries myself.

File: src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c [hunk 3]
```diff
@@ -61,7 +61,7 @@ static u32 D_808B9104[] = {
     0,
 };
 
-static BgSpot18ObjInitFunc D_808B910C[] = {
+static BgSpot18ObjInitFunc sInitFuncs1[] = {
```
[the remark this answers is not in the document]
Comment (Dragorn421) [reply src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c:64]: When I said
> I would leave it unnamed until more functions are named in this actor
I meant the name should probably stay `D_808B910C` unless you can think of a better name

[the remark this answers is not in the document]
Comment (Meowsy) [reply src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c]: I somehow managed to not stage *only* the lines related to `D_808B910C`. I don't even understand how.

**Review by Dragorn421 — APPROVED** [review]

**Comment by fig02** [comment]
Im a little wary of naming every empty function in an actor as "do nothing". The ones that are called do nothing currently are done so cause it makes sense in the context of the actor.
For example, in darkmeiro, its called "noop" as in no operation on purpose-- the function callbacks arent really actions in this case, they are basically update functions. So no operation is more appropriate than do nothing

I would prefer if they got named with the whole actor being documented so we are sure the name makes sense in that context

**Comment by fig02** [comment]
I'm going to close this for now, for the reasons mentioned above. I hope this doesn't discourage you from getting into doing documentation work though. Its just pretty important for us that this stuff is done at the system level so that all context can be taken into account.
I recommend picking an actor to document in-full instead of naming a bunch of similar functions in different actors. Let me know if you have any questions are reach out on the discord.

**Pull request closed without merging**